setup.py

```diff
@@ -220,6 +220,9 @@
                             "yt/geometry/",
                             "yt/utilities/lib/"],
               depends=glob.glob("yt/frontends/artio/artio_headers/*.c")),
+    Extension("yt.frontends.amrvac.tests.parfiles",
+              "yt/frontends/amrvac/test/sample_parfiles/*.par",
+    )
```

Comment:
This isn't correct, you're not building a C extension. Let's chat about this in a lower-latency context.

Reply:
force-pushed so there no trace of this blatant mistake 🙈